docs/framework/unmanaged-api/debugging/ixclrdatamodule-interface.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "IXCLRDataModule Interface"
+ms.date: "01/15/2019"
+api.name:
+  - "IXCLRDataModule"
+api.location:
+  - "mscordacwks.dll"
+api.type:
+  - "COM"
+f1.keywords:
+  - "IXCLRDataModule Interface"
+helpviewer.keywords:
+  - "IXCLRDataModule Interface [.NET Framework debugging]"
+topic_type:
+  - "apiref"
+author: "cshung"
+ms.author: "andrewau"
+---
+# IXCLRDataModule Interface
+Provide methods for querying information about a loaded module.
+
+## Methods
+|Method                    |Description                               |
+|--------------------------|------------------------------------------|
```

Comment:
The members should link to the member documentation. The "Get" in each member definition should be "Gets".